metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_connection.py

```diff
@@ -2,16 +2,23 @@
 import os
 from typing import Any, Dict, Optional
 
+from google.api_core.client_info import ClientInfo
 from google.cloud import bigquery, datacatalog_v1, resourcemanager_v3
 from google.cloud.logging_v2.client import Client as GCPLoggingClient
 from pydantic import Field, PrivateAttr
 
+from datahub._version import nice_version_name
 from datahub.configuration.common import ConfigModel
 from datahub.ingestion.source.common.gcp_credentials_config import GCPCredential
 
 logger = logging.getLogger(__name__)
 
 
+def _get_bigquery_client_info() -> ClientInfo:
+    """Get ClientInfo with DataHub user-agent for BigQuery client identification"""
+    return ClientInfo(user_agent=f"datahub/{nice_version_name()}")
```

Comment:
use `__version__` and not nice_version_name

`nice_version_name` is for human-readable strings